src/sentry/replays/testutils.py

```diff
@@ -103,8 +103,12 @@ def mock_replay(
                             "transaction": kwargs.pop("title", "Title"),
                         },
                         "urls": kwargs.pop("urls", []),
-                        "error_ids": kwargs.pop("error_ids", []),
-                        "trace_ids": kwargs.pop("trace_ids", []),
+                        "error_ids": kwargs.pop(
+                            "error_ids", ["a3a62ef6-ac86-415b-83c2-416fc2f76db1"]
```

Comment:
these shouldn't have dashes in them, and fails locally if you remove them, but this is working in prod 🤔  -- at any rate, i believe i know what the fix is and will make it snuba side.